src/index.html

```diff
@@ -31,7 +31,155 @@
     </nav>
   </header>
 
-  <main></main>
+  <main>
+    <div class="p-keyvisual">
+      <p class="p-keyvisual-text"><img src="./img/picture/keyvisual-text.png" alt="見出しバナー"></p>
+      <p class="p-keyvisual-title"><img src="./img/picture/keyvisual-title.png" alt="医療求人バナー"></p>
+      <div class="box">
+        <p>現在の求人掲載数：<span class="number">2,320</span>件</p>
+      </div>
+    </div>
+    
+    <div class="p-jobsearch">
+      <hgroup>
+        <h2>お仕事検索</h2>
+        <p>働きたい職種・地域・希望給与を入れて検索！</p>
+      </hgroup>
+      <div class="select-box">
+        <div class="box_wrapper">
+          <select name="job"  id="select" class="select">
+            <option value="job1"selected>医療事務</option>
```

Comment:
スペースを開けるようにお願いします
他の箇所の修正もお願いします

```suggestion
            <option value="job1" selected>医療事務</option>
```

Reply:
スペースを変更しました。
6dcee299a7f6385aa8b967bd33169190692e4dae